.github/workflows/ci.yml

```diff
@@ -171,14 +171,20 @@ jobs:
     steps:
       - uses: taiki-e/checkout-action@b13d20b7cda4e2f325ef19895128f7ff735c0b3d # v1.3.1
 
+      - name: Setup Rust
+        uses: oxc-project/setup-rust@1ff88fdaffd6ae35c0fb32ccc159340b37d0beac # v1.0.7
```

Comment:
- Check `resuable-node-test.yml` to see how to setup enviroment for building

- Rolldown depends on `@rolldown/pluginutils`.